lib/test/test_hamburger.rb

```diff
@@ -0,0 +1,136 @@
+require_relative '../../lib/cdo/hamburger'
+require 'minitest/autorun'
+
+class HamburgerTest < Minitest::Test
+  def test_level_teacher_en
+    visibility = Hamburger.get_visibility({level: true, user_type: "teacher", language: "en"})
+
+    assert visibility[:hamburger_class]         == "show-always"
+    assert visibility[:show_teacher_options]    == "show-always"
+    assert visibility[:show_student_options]    == "hide-always"
+    assert visibility[:show_signed_out_options] == "hide-always"
+    assert visibility[:show_pegasus_options]    == "show-always"
+    assert visibility[:show_help_options]       == "show-always"
+  end
+
+  def test_level_teacher_nonen
+    visibility = Hamburger.get_visibility({level: true, user_type: "teacher", language: "fr"})
```

Comment:
It doesn't really matter but if you're not picky, I'd rather test with 'es' (Spanish) since that's our next top language. It gets higher visibility etc. so if there are issues with just a particular language I'd rather catch with Spanish sooner.